src/utils/getAvatarUrl.ts

```diff
@@ -0,0 +1,3 @@
+export function getAvatarUrl(name: string) {
+  return `/images/users/${name}.jpeg`;
+}
```

Comment:
Could you add a check here to see if the file exists, and if not, return 'unnamed.jpeg'? In that way, the images will load smoother and we won't get a bunch of `404: File not found` errors if something is missing. 😁

Reply:
I am not sure if this is possible, maybe on build time? So we would need to generate a list of available images and read it on runtime. I've rewritten the UserAvatar a bit and hope this fixes some of the jankiness :) What do you think?